python/ray/projects/examples/open-tacotron/.rayproject/project.yaml

```diff
@@ -0,0 +1,17 @@
+# This file is generated by `ray project create`
+
+name: open-tacotron
+description: "A TensorFlow implementation of Google's Tacotron speech synthesis with pre-trained model (unofficial)"
+repo: https://github.com/keithito/tacotron
+
+cluster: .rayproject/cluster.yaml
+
+environment:
+  requirements: requirements.txt
```

Comment:
this file doesn't exist?

Reply:
ah i see, this is supposed to exists in clone repo

Reply:
yes :)